Mathlib/Analysis/Normed/Unbundled/SpectralNorm.lean

```diff
@@ -841,39 +880,111 @@ def nontriviallyNormedField [CompleteSpace K] : NontriviallyNormedField L where
     ⟨algebraMap K L x, hx.trans_eq <| (spectralNorm_extends _).symm⟩
 
 /-- `L` with the spectral norm is a `normed_add_comm_group`. -/
-def normedAddCommGroup [CompleteSpace K] : NormedAddCommGroup L := by
+def normedAddCommGroup : NormedAddCommGroup L := by
   haveI : NormedField L := normedField K L
   infer_instance
 
 /-- `L` with the spectral norm is a `seminormed_add_comm_group`. -/
-def seminormedAddCommGroup [CompleteSpace K] : SeminormedAddCommGroup L := by
+def seminormedAddCommGroup : SeminormedAddCommGroup L := by
   have : NormedField L := normedField K L
   infer_instance
 
 /-- `L` with the spectral norm is a `normed_space` over `K`. -/
-def normedSpace [CompleteSpace K] : @NormedSpace K L _ (seminormedAddCommGroup K L) :=
-  letI _ := seminormedAddCommGroup K L
-  {(inferInstance : Module K L) with
-    norm_smul_le r x := by
-      change spectralAlgNorm K L (r • x) ≤ ‖r‖ * spectralAlgNorm K L x
-      exact le_of_eq (map_smul_eq_mul _ _ _)}
+def normedSpace : @NormedSpace K L _ (seminormedAddCommGroup K L) :=
+   letI _ := seminormedAddCommGroup K L
+   {(inferInstance : Module K L) with
+     norm_smul_le r x := by
+       change spectralAlgNorm K L (r • x) ≤ ‖r‖ * spectralAlgNorm K L x
+       exact le_of_eq (map_smul_eq_mul _ _ _)}
 
 /-- The metric space structure on `L` induced by the spectral norm. -/
-def metricSpace [CompleteSpace K] : MetricSpace L := (normedField K L).toMetricSpace
+def metricSpace : MetricSpace L := (normedField K L).toMetricSpace
 
 /-- The uniform space structure on `L` induced by the spectral norm. -/
-def uniformSpace [CompleteSpace K] : UniformSpace L := (metricSpace K L).toUniformSpace
+def uniformSpace : UniformSpace L := (metricSpace K L).toUniformSpace
 
 /-- If `L/K` is finite dimensional, then `L` is a complete space with respect to topology induced
   by the spectral norm. -/
-instance (priority := 100) completeSpace [CompleteSpace K] [h_fin : FiniteDimensional K L] :
+instance (priority := 100) completeSpace [h_fin : FiniteDimensional K L] :
     @CompleteSpace L (uniformSpace K L) := by
   letI := (normedAddCommGroup K L)
   letI := (normedSpace K L)
   exact FiniteDimensional.complete K L
 
+omit [Algebra.IsAlgebraic K L]  in
+/-- Given an algebraic tower of fields `E/L/K` and an element `x : L` whose minimal polynomial `f`
+  over `K` splits into linear factors over `E`, the `degree(f)`th power of the spectral norm of `x`
+  is equal to the product of the `E`-valued roots of `f`. -/
```

Comment:
What the statement says is that ... is equal to the *spectral norm of* the product ...